wave_kinematics/src/quadrotor.cpp

```diff
@@ -0,0 +1,435 @@
+#include "wave/kinematics/quadrotor.hpp"
+
+namespace wave {
+
+// ATTITUDE CONTROLLER
+AttitudeController::AttitudeController(void) {
+    this->dt = 0.0;
+    this->outputs << 0.0, 0.0, 0.0, 0.0;
+
+    this->roll_controller = PID(200.0, 0.5, 10.0);
+    this->pitch_controller = PID(200.0, 0.5, 10.0);
+    this->yaw_controller = PID(200.0, 0.5, 10.0);
+}
+
+Vec4 AttitudeController::update(Vec4 setpoints, Vec4 actual, double dt) {
```

Comment:
Sorry this is my bad this code was rushed in as a side project for steve, and I didn't give it thought to  explan the code well enought. Still, no excuses, will add documentation soon.